back/spec/acceptance/initiative_status_changes_spec.rb

```diff
@@ -124,34 +126,63 @@
         end
       end
 
-      context 'when the review feature is fully active and the status change is to the proposed status' do
-        before do
-          SettingsService.new.activate_feature! 'initiative_review'
+      # If the review feature is off, the initiative_status_change with code 'proposed'
+      # is created at the model level when the initiative is published.
+      # Thus, this POST request can only happen when the review feature is on.
+      context 'when the the status change is to the proposed status' do
+        let(:new_initiative) { create(:initiative) }
+        let!(:_initiative_status_change) do
+          create(
+            :initiative_status_change,
+            initiative: new_initiative,
+            initiative_status: @status_review_pending
+          )
+        end
+
+        let(:initiative_id) { new_initiative.id }
+        let(:initiative_status_id) { @status_proposed.id }
+
+        example 'creation results in locked editing' do
+          expect(Initiative.find(initiative_id).editing_locked).to be false
 
-          configuration = AppConfiguration.instance
-          configuration.settings['initiatives']['require_review'] = true
-          configuration.save!
+          do_request
+          assert_status 201
+          expect(Initiative.find(initiative_id).editing_locked).to be true
+        end
+
+        example "creation results in the logging of an Initiative 'proposed' activity" do
+          expect { do_request }
+            .to have_enqueued_job(LogActivityJob)
+            .with(instance_of(Initiative), 'proposed', @user, instance_of(Integer))
+            .exactly(1).times
```

Comment:
I would probably combine these two tests into one for the sake of performance. Our BE tests is the slowest CI part atm.

Reply:
IMO, tests should really test only one thing, and also I want to use the example description to clarify the behaviour of the `proposed` activity.